Assets/CesiumForUnitySamples/Scenes/VR02_CesiumMetadata.unity

```diff
@@ -0,0 +1,2874 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
```

Comment:
One more thing -- the "Getting Started" text is the exact same as the first VR level. Since we already have build instructions for the first level, let's have the second level's description be more specific about what it's showing. Feel free to reference the other scenes' text for inspiration.

We also need a section for this in the README.